README.md

```diff
@@ -1,9 +1,9 @@
-# Material Design for Angular 2
+# Material Design for Angular
 [![npm version](https://badge.fury.io/js/%40angular%2Fmaterial.svg)](https://www.npmjs.com/package/%40angular%2Fmaterial)
 [![Build Status](https://travis-ci.org/angular/material2.svg?branch=master)](https://travis-ci.org/angular/material2)
 [![Gitter](https://badges.gitter.im/angular/material2.svg)](https://gitter.im/angular/material2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-This is the home for the Angular team's Material Design components built on top of Angular 2.
+This is the home for the Angular team's Material Design components built for **Angular version 2 onwards**.
```

Comment:
If I understood them properly, @igorminar as well as @StephenFluin said to me one should mention "version 2" and not just "v2" to make it more explicit. More correct even, "Angular" and "version" should not be in the same sentence. So it should maybe be "for version 2+". I just thought that would create confusion with material 1 & 2...